lib/stdlib/doc/src/timer.xml

```diff
@@ -249,6 +254,11 @@
           or suspends the process forever if <c><anno>Time</anno></c> is the
           atom <c>infinity</c>. Naturally, this
           function does <em>not</em> return immediately.</p>
+        <note>
+          <p>Before OTP 25, <c>timer:sleep/1</c> did not accept integer
+            timeout values greater than <c>16#ffffffff</c>, that is, <c>2^64-1</c>.
```

Comment:
```suggestion
            timeout values greater than <c>16#ffffffff</c>, that is, <c>2^32-1</c>.
```

Reply:
Otherwise the documentation looks good to me.

Reply:
Oh, of course 😅